cmd/opm/alpha/bundle/extract.go

```diff
@@ -55,9 +56,13 @@ func runExtractCmd(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	gzip, err := cmd.Flags().GetBool("gzip")
+	if err != nil {
+		return err
+	}
 
 	loader := configmap.NewConfigMapLoaderForDirectory(configmapName, namespace, manifestsDir, kubeconfig)
-	if err := loader.Populate(datalimit); err != nil {
+	if err := loader.Populate(datalimit, gzip); err != nil {
```

Comment:
I've been debating this with myself too, and didn't reach a clear conclusion :-) Eventually I chose to pass this parameter together with `datalimit`, since they're somehow related in dealing with bundle size. That was my logic at least. Anyway, no problem to change to a loader attribute.